civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +65,198 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .contact-id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom:0px;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 100px;
+    position: relative;
+  }
+
+  .views-field-image-URL {
+    left: 0px;
+    position: absolute;
+    top: 0px;
+  }
+
+  .no-user-image {
```

Comment:
the current my details page already handles the default image, so why do we need to do it all over again?

Reply:
I'm not using the same view because dashboard already needs this view. It is a different view in purpose to avoid touching anything else but My Details page.